os&sys/ls.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/python3
```

Comment:
Почти! Эта строка "скажет" командной оболочке использовать интерпретатор который лежит в `/usr/bin`. Для тоо же, чтобы использовать питон из текущего активного окружения нужно написать вот такую штуку
```suggestion
#!/usr/bin/env python
```